includes/utils.php

```diff
@@ -1121,3 +1123,30 @@ function post_args_allow_list( $post_args ) {
 
 	return array_intersect_key( $post_args, array_flip( $allowed_post_keys ) );
 }
+
+/**
+ * Make a remote GET request.
+ *
+ * Wrapper function for wp_remote_get() and vip_safe_wp_remote_get(). The order
+ * of parameters differs from vip_safe_wp_remote_get() to promote the arguments array
+ * to the second parameter.
+ *
+ * See {@see http://developer.wordpress.org/reference/classes/WP_Http/request/ WP_Http::request} for $args defaults.
+ *
+ * @param  string $url       The URL to request.
+ * @param  array  $args      Optional. An array of arguments to pass to wp_remote_get()/vip_safe_wp_remote_get().
+ * @param  mixed  $fallback  Optional. Fallback value to return if the request fails. Default false. VIP only.
+ * @param  int    $threshold Optional. The number of fails required before subsequent requests automatically return the fallback value. Defaults to 3, with a maximum of 10. VIP only.
+ * @param  int    $timeout   Optional. The timeout for WP VIP requests. Default 3, 1-5 valid. VIP only, use $args for others.
+ * @param  int    $retries   Optional. The number of retries to attempt. Deafult 10. VIP only.
```

Comment:
May be worth noting here that this value can't be less than 10, at least according to VIP's documentation (and they actually set the default to 20, may be worth matching that?)

Reply:
> May be worth noting here that this value can't be less than 10, at least according to VIP's documentation (and they actually set the default to 20, may be worth matching that?)

I'll update the docs

All the instances in the code base set this to ten, thus the lower default. It may be worth changing it but let's do so in a follow up to keep the code as close to unchanged as possible.